src/test/java/picard/fingerprint/CrosscheckReadGroupFingerprintsTest.java

```diff
@@ -0,0 +1,312 @@
+package picard.fingerprint;
+
+import htsjdk.samtools.metrics.MetricsFile;
+import org.testng.Assert;
+import org.testng.annotations.BeforeTest;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+import picard.vcf.SamTestUtils;
+
+import java.io.*;
+import java.lang.reflect.Field;
+import java.nio.charset.Charset;
+import java.nio.file.Files;
+import java.util.*;
+import java.util.stream.Stream;
+
+import static picard.fingerprint.FingerprintIdDetails.multipleValuesString;
+
+/**
+ * Tests for CrosscheckReadGroupFingerprints
+ */
+public class CrosscheckReadGroupFingerprintsTest {
+
+    private final static File TEST_DIR = new File("testdata/picard/fingerprint/");
```

Comment:
Should be consistent here with the ordering of modifiers. I think the preferred order is `private final static`. There is an IDE setting to highlight this and fix it for you.